lua/nvim-tree/appearance.lua

```diff
@@ -196,8 +196,15 @@ function M.setup()
 
   -- hard link override when legacy only is present
   for from, to in pairs(LEGACY_LINKS) do
-    local hl_from = vim.api.nvim_get_hl(0, { name = from })
-    local hl_to = vim.api.nvim_get_hl(0, { name = to })
+    local hl_from
+    local hl_to
+    if vim.fn.has "nvim-0.8" == 1 then
```

Comment:
Unfortunately we'll have to invert this condition: `nvim-0.8` is true when using 0.9

I discovered this as `nvim__get_hl_defs` has been completely removed from 0.9. So much for non-breaking changes...